modules/serverless-rn-1-33-0.adoc

```diff
@@ -0,0 +1,46 @@
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-33-0_{context}"]
+= Red Hat {ServerlessProductName} 1.33
+
+{ServerlessProductName} 1.33 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-33-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.12.
+* {ServerlessProductName} now uses Knative Eventing 1.12.
+* {ServerlessProductName} now uses Kourier 1.12.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
+* The `kn func` CLI plugin now uses `func` 1.13.
+
+* When specifying Kourier for Ingress and using `DomainMapping`, the TLS for OpenShift Route is set to passthrough, and TLS is handled by the Kourier Gateway. Beginning with Serverless 1.31, you can specify the enabled cipher suites on the side of the Kourier Gateway.
+
+* The `NamespacedKafka` annotation is now deprecated. Use the standard Kafka broker without data plane isolation instead.
+
+* When enabling the automatic `EventType` auto-creation, you can now easily discover events available within the cluster. This functionality is available as a developer preview 
+
+* The `kn` event plugin for the Knative CLI (kn) is now generally available. The `kn` event plugin lets you send events directly from your command line to various destinations, streamlining your development workflow.
+
+* The new trigger filters feature is now available as a Technology Preview. The new trigger filters are now enabled by default. It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
+
+* You can now explore the Knative Eventing monitoring dashboards directly within the *Observe* tab of the Developer view in the OpenShift Developer Console. 
+
+* {FunctionsProductName} now supports Go function container and is now available as a Technology Preview (TP) feature.
+
+* You can now use the Custom Metrics Autoscaler Operator to autoscale Knative Eventing sources for Apache Kafka sources (`KafkaSource`). This functionality is available as a developer preview, offering enhanced scalability and efficiency for Kafka-based event sources within Knative Eventing.
+
+[id="known-issues-1-33-0_{context}"]
+== Known issues
+
+* Due to different mount point permissions, direct upload on a cluster build does not work on IBM zSystems (s390x) and IBM Power (ppc64le).
+
+* Building and deploying a function using Podman version 4.6 fails with the `invalid pull policy "1"` error.
++
+To work around this issue, use Podman version 4.5.
+
+* When upgrading from version 1.32.0 to 1.32.1, note that all Pods of Knative Services will restart. Ensure that Pods start correctly by verifying that images can be pulled and other dependencies are met to prevent connection drops.
```

Comment:
Is this note valid? We're documenting 1.33.0

Reply:
@mgencur I suppose this is covered here because there were no 1.32.1 release notes, and a reader would not look for a `1.32.0-to-1.32.1` note in 1.32.0 release notes: we don't expect post-1.32.0 stuff to be in 1.32.0 notes.

If during a `1.32.0-to-1.33.0` upgrade this is not relevant, we can remove this, just tell me.

Reply:
OK. We should remove this note. The issue was fixed in 1.32.1 and the upgrade from 1.32.1 to 1.33.0 works as expected. This is not an issue anymore for 1.33.

Reply:
@rh-max looks like this feedback was not implemented. I still see line 46 in the PR